src/azure-cli/azure/cli/command_modules/util/_help.py

```diff
@@ -22,6 +22,12 @@
 
 
     For passing JSON in PowerShell, see https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md
+
+
+    [WARNING] Depending on the REST API you call, the output of this command may include secrets that you must protect.
```

Comment:
Not all `az rest` commands show secrets, so I made a small change here, compared to `OUTPUT_WITH_SECRET`.